ibis_substrait/compiler/translate.py

```diff
@@ -906,3 +906,25 @@ def aggregation(
     )
 
     return stalg.Rel(aggregate=aggregate)
+
+
+@translate.register(ops.SimpleCase)
+def _simple_case(
+    op: ops.SimpleCase,
+    expr: ir.TableExpr,
+    compiler: SubstraitCompiler,
+    **kwargs: Any,
+) -> stalg.Expression:
+    # the field names for an `if_then` are `if` and `else` which means we need
+    # to pass those args in as a dictionary to not run afoul of SyntaxErrors`
+    _ifs = []
+    for case, result in zip(op.cases, op.results):
+        _if = {
+            "if": translate(op.base == case, compiler),
```

Comment:
Probably wanna thread `**kwargs` thought the `translate` calls.

Reply:
:+1: done